lib/live_debugger/components.ex

```diff
@@ -142,9 +142,8 @@ defmodule LiveDebugger.Components do
       {show_collapsible_assign(@open)}
     >
       <summary
-        class={[
-          "block flex items-center cursor-pointer" | List.wrap(@label_class)
-        ]}
+        id={@id <> "-summary"}
+        class={["block flex items-center cursor-pointer" | List.wrap(@label_class)]}
```

Comment:
I think block and flex exclude each other - they are both values for `display` property